opcua/ua/uautils.py

```diff
@@ -0,0 +1,8 @@
+from opcua.ua import ObjectIds, ObjectIdNames
+from opcua.ua.uaprotocol_auto import ExtensionClasses
+
+
+def register_extension_object(object_factory):
+    setattr(ObjectIds, "{}_Encoding_DefaultBinary".format(object_factory.__name__), object_factory.DEFAULT_BINARY)
+    ObjectIdNames[object_factory.DEFAULT_BINARY] = object_factory.__name__
+    ExtensionClasses[object_factory.DEFAULT_BINARY] = object_factory
```

Comment:
we already have a file called ua_utils.py under common. Maybe you could put that file in uatypes.py or ua_prototcol_hans.py